### ISSUE TITLE
Use Webpack Hash

### ISSUE THREAD

**Issue by NumerHero** [SimenB/add-asset-html-webpack-plugin#58]
now I make a Dll and output.filename with hash:5 to bundle vendors

```
const vendors = [
    'react', 'react-router', 'react-dom', 'classnames', 'antd', 'whatwg-fetch', 'redux', 'react-redux'
];
 entry: {
        vendors: vendors
    },
output: {
        path: './dist',
        publicPath: publicPath,
        filename: '[name].[hash:5].js',
        library: '[name]'
    },
```

However I want use add-asset-html-webpack-plugin to inject a script with vendors.[hash:5].js

I try the hash option and find it didnt match with webpack hash

now I solve this problem with glob and add-asset-html-webpack-plugin

```
// 提取vendors[hash:5]
const VENDORS_PATH = glob.sync('./dist/vendors.*.js');
```

meanwhile

```
 new AddAssetHtmlPlugin({filepath: require.resolve(VENDORS_PATH)})
```

However I thought It too trouble， so I thought could you please strengthen this plugin to support webpack hash?

**Comment by NumerHero** [comment]
It's best you can integrate the glob and ExtractTextPlugin

**Comment by SimenB** [comment]
Changing the hash options is kinda #56, this goes one step further though. I'll have to check how webpack itself handles hash, maybe I can use their api (with it being in the filename and not as a separate option).

Handling a glob can be done though! Could you send a PR?

To give you a start: use `globby` for promise glob (`await` it). Check if the passed string has a `*` in it, and if so pass it to globby.
Not sure how to handle multiple matches, though. Thoughts?

**Comment by NumerHero** [comment]
Sorry ... give me some time

**Comment by NumerHero** [comment]
Hi, I make a Realization @SimenB

**Comment by JimmyLv** [comment]
@SimenB Have you merge the functionality?

**Comment by SimenB** [comment]
@JimmyLv See #78 for open PR

**Comment by SimenB** [comment]
Glob was merged in #78, and hash is tracked in #56. Closing
